lib/components/form/settings-preview.js

```diff
@@ -40,7 +46,22 @@ class SettingsPreview extends Component {
 
     return (
       <div className='settings-preview' onClick={this.props.onClick}>
-        <div className='summary'>Transit Options<br />&amp; Preferences</div>
+        <div className='summary'>
+          {// If settings preview is multiple lines, split
+            Array.isArray(messages.label)
+              ? <span>
+                {messages.label[0]}
+                <br />
+                {messages.label[1]}
+              </span>
+              : <span
+                // Increase line height if number of lines is 1 for vertical
+                // centering.
+                style={{lineHeight: 2.6}}>
+                {messages.label}
+              </span>
```

Comment:
Could this work? https://stackoverflow.com/a/55466235

Reply:
Good suggestion. It might be tricky to get the styling right on that though (the line height = 2.6 is important for centering vertically and might be hard to detect without the array).